### PR TITLE
add watson studio

### PR DESCRIPTION
**Pull request by hiltol**

### CODE REVIEW

File: infracost-usage-example.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ resource_type_default_usage:
     wd_queries: 11000
     wd_custom_models: 4
     wd_collections: 301
+    data-science-experience_CAPACITY_UNIT_HOURS: 1
```
Comment (luisarojas) [inline infracost-usage-example.yml:80]: should we name this something like `data_science_experience_capacity_unit_hours` to follow the same naming conventions?

**Review by luisarojas — APPROVED** [review]

**Comment by luisarojas** [comment]
https://github.ibm.com/cloud-finops/issues/issues/189

File: infracost-usage-example.yml (hunk 1 above)
Comment (hiltol) [reply infracost-usage-example.yml:80]: trying something new with a POC to integrate with Demis' work. The idea would be to adopt a new naming convention for these that would align with the names the company uses for rated usage metrics

Comment (luisarojas) [reply infracost-usage-example.yml:80]: Ohh I see, understood

**Pull request merged**